python/ray/workflow/tests/test_recovery.py

```diff
@@ -103,12 +103,6 @@ def test_dedupe_downloads_list(workflow_start_regular):
         assert get_objects_count == 5
 
 
-@pytest.mark.parametrize(
```

Comment:
Do we still need the changes to this test?

Reply:
hold on yeah i just messed up reverting this change.